src/client/app/components/MultiCompareChartComponent.tsx

```diff
@@ -4,10 +4,11 @@
 
 import * as React from 'react';
 import CompareChartContainer from '../containers/CompareChartContainer';
+import { Entity } from '../containers/MultiCompareChartContainer';
 
 interface MultiCompareChartProps {
-	selectedMeters: number[];
-	selectedGroups: number[];
+	selectedMeters: Entity[];
```

Comment:
Entity is not super descriptive. I would prefer if this was called `CompareEntity`.